src/strategies/api/index.ts

```diff
@@ -13,9 +13,14 @@ export async function strategy(
   snapshot
 ) {
   let api_url = options.api + '/' + options.strategy;
-  api_url += '?network=' + network;
+  api_url += '?';
+  if (options.additionalParameters) {
+    api_url += options.additionalParameters + '&'
+  }
+  api_url += 'network=' + network;
   api_url += '&snapshot=' + snapshot;
   api_url += '&addresses=' + addresses.join(',');
+  api_url += options.additionalParameters;
```

Comment:
Why `additionalParameters` are adding up to api_url twice? 🤔

Reply:
oh sorry line #23 had to be removed. at start it just added it in the end, but for better formet URL i thought to put it infornt instead